SysKit.XCellKit/SysKit.XCellKit.csproj

```diff
@@ -10,5 +10,8 @@
 	<ItemGroup>
 		<PackageReference Include="DocumentFormat.OpenXml" Version="2.14.0" />
 		<PackageReference Include="Microsoft.Windows.Compatibility" Version="5.0.2" />
```

Comment:
Microsoft.Windows.Compatibility is okay? does it work on linux? or just builds and has a runtime issue?

Reply:
I removed this package. Also I have tested it on windows and linux, everything works fine without it.